q2lint.py

```diff
@@ -30,6 +30,18 @@ def main():
             header = ''.join(line for _, line in zip(range(7), filehandle))
             if header != HEADER:
                 errors.append('Invalid header: %s' % filepath)
+            if filepath.name != 'setup.py':
+                continue
+            text = filehandle.read()
+            if 'install_requires' in text:
```

Comment:
We were planning to allow the following "success" cases:

- `install_requires` isn't in setup.py
- `install_requires=[]` is in setup.py

Can you update your code to accept the second case as a "success"? Most of the q2 packages specify an empty `install_requires` so we might as well support that to avoid having to update those packages.

I think you can accomplish this without regexes:

```python
if 'install_requires' in text and 'install_requires=[]' not in text:
    errors.append("Package dependencies should be stored in a conda recipe instead of setup.py `install_requires`.")
```